docs/configuration/optional-settings.md

```diff
@@ -135,6 +135,22 @@ Enforcement of unique IP space can be toggled on a per-VRF basis. To enforce uni
 
 ---
 
+## GITHUB_REPOSITORY_API
+
+Default: 'https://api.github.com/repos/netbox-community/netbox'
+
+The releases of this repository are checked to detect new releases, which are shown on the home page of the web interface. You can change this to your own fork of the NetBox repository, or set it to `None` to disable the check.
```

Comment:
We should call out that the URL provided MUST adhere to GitHub's API. (I haven't checked but I assume this would not work against GitLab, for example.)